src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/order_actions.html.twig

```diff
@@ -81,44 +94,31 @@
       </div>
     </form>
   {% endif %}
+
   {% include '@PrestaShop/Admin/Sell/Order/Order/Blocks/View/extra_order_button_actions.html.twig' %}
+
   {% if merchandiseReturnEnabled %}
     {% if orderForViewing.isDelivered() %}
-      <div class="form-inline d-inline-block form-horizontal ml-2">
-        <div class="input-group">
-          <button class="btn btn-action return-product-display" type="button"{% if not orderForViewing.isRefundable() %} disabled{% endif %}>
-            <i class="material-icons">swap_horiz</i>
-            {{ 'Return products'|trans({}, 'Admin.Orderscustomers.Feature') }}
-          </button>
-        </div>
-      </div>
+      <button class="btn btn-action return-product-display" type="button"{% if not orderForViewing.isRefundable() %} disabled{% endif %}>
```

Comment:
I believe I added this because there was a small offset compared with the buttons on the left of this block This was spotted by @Robin-Fischer-PS with the accurate use of a ruler!! Like a real life ruler!!
If you managed to get rid of all those extra divs and keep all the buttons aligned that's great!! Just be careful as it was a really small offset (1px)

Reply:
Ahahahah, it should be fine, it looks aligned :)